src/pages/Services.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import { get } from 'lodash';
+import data from '__tmpdata/services';
+import FormFeedback from 'components/FormFeedback';
+var axios = require('axios');
+
+class ServicesIndex extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      body: ''
+    };
+  }
+
+  componentDidMount() {
+    axios
```

Comment:
as we find ourselves writing more axios calls, we could move whatever is common into a utility module. Something like `getDataFromJoplin(pageId)`. Then we can include any functions we need in our components from that utility module and push more logic out of the component.